common.mk

```diff
@@ -320,6 +329,8 @@ filelist-tee-common: optee-client xtest helloworld
 									>> $(fl)
 	@echo "slink /lib/libteec.so.1 libteec.so.1.0 755 0 0"			>> $(fl)
 	@echo "slink /lib/libteec.so libteec.so.1 755 0 0" 			>> $(fl)
+	@echo "dir /etc/udhcp 755 0 0" 			>> $(fl)
+	@echo "file /etc/udhcp/simple.script $(ROOT)/busybox/examples/udhcp/simple.script 755 0 0" >> $(fl)
```

Comment:
I still think it's better to put this into the gen_rootfs, since other files from busybox are included there directly. Here we typically only include non-busybox binaries etc.